src/main/java/oreo/exception/IllegalDateTimeException.java

```diff
@@ -0,0 +1,16 @@
+package oreo.exception;
+
+import java.time.DateTimeException;
+
+public class IllegalDateTimeException extends Exception {
+    private String msg;
+
+    public IllegalDateTimeException(String msg) {
+        this.msg = msg;
+    }
+
+    @Override
+    public String getMessage() {
+        return msg;
+    }
+}
```

Comment:
For `Exception` subclasses, I believe you can simply use `message` which is already available as part of the `Exception` superclass?

```java
public class IllegalDateTimeException extends Exception {
    public IllegalDateTimeException(String msg) {
        super(msg);
    }
}
```

With this, `getMessage` _just works._